README.md

```diff
@@ -9,10 +9,16 @@ This is a temporary website linking some of the demos and tools we build to supp
 
 [Fruit identification using Arduino and TensorFlow](https://blog.arduino.cc/2019/11/07/fruit-identification-using-arduino-and-tensorflow/)
 
+[Use the dashboard using the Arduino Nicla Sense ME](http://docs.arduino.cc/tutorials/nicla-sense-me/web-ble-dashboard)
 
 ## BLE Sense Demo Dashboard
 
 [BLE Sense Demo - Arduino sketch](https://create.arduino.cc/editor/dpajak/e4128780-d76d-438f-97a4-541740cb0473/preview)
 
 [BLE Sense Demo - Web Dashboard](https://arduino.github.io/ArduinoAI/BLESense-test-dashboard) (Chrome only)
 
+## Nicla Sense ME Demo Dashboard
+
+[Nicla Sense ME BLE Demo - Arduino sketch](https://create.arduino.cc/editor/pmarquinez/b0102b0a-1d8b-4b4a-a53d-781b896d87f7/preview)
```

Comment:
@marqdevx Can we use the "official" Arduino Cloud account to host the sketch?

Reply:
Fixed